public/data/Abdus_Salam.json

```diff
@@ -16,13 +16,12 @@
     },
     {
       "name": "I'm on Telegram most of the time",
-      "url": "t.me/aladdin_02",
+      "url": "https://t.me/aladdin_02",
       "icon": "telegram"
     },
     {
       "name": "See my awesome tweets on Twitter",
-      "url": "https://twitter.com/abdussalam02",
-      "icon": "twitter"
+      "url": "https://twitter.com/abdussalam_02",
```

Comment:
Why you removed the icon ?
Also the GitHub actions complaining on the trailing comma you left after the URL as you removed the icon. Fix that